xml/art_jeos_quickstart.xml

```diff
@@ -216,6 +216,35 @@
     </para>
      </listitem>
     </varlistentry>
+    <varlistentry>
+     <term>systemd-coredump disabled</term>
+     <listitem>
+      <para>
+       Because of the small amount of memory and disk space available by default on JeOS images,
+       systemd-coredump is disabled by default on JeOS images.
+       If your usage scenario requires to collect kernel core dumps for analyzing application
+       crashes:
+       <itemizedlist>
+        <listitem>
+         <para>
+	  Install the <package>systemd-coredump</packages> package,
+         </para>
+        </listitem>
+        <listitem>
+         <para>
+         Set <link xlink:href="https://www.freedesktop.org/software/systemd/man/systemd-coredump.html">kernel.core_pattern</link>
+         in <link xlink:href="https://www.suse.com/support/kb/doc/?id=000018634">sysctl</link>
+         </para>
+        </listitem>
+        <listitem>
+         <para>
+          Check out our <link xlink:href="https://documentation.suse.com/sles/15-SP3/single-html/SLES-tuning/#cha-tuning-systemd-coredump">systemd-coredump documentation</link>.
```

Comment:
This shouldn't link to a fixed version, if possible.

Reply:
I'm editing the SP3 JeOS quickstart, so it is fine to me to point to the SP3 documentation. But in general I agree with your statement but I don't think we can do that.

Reply:
@dmpop what do you think about this?